content/en/docs/tasks/observability/distributed-tracing/configurability/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Configurability (Experimental)
+description: How to configure tracing options (experimental).
+weight: 60
+keywords: [telemetry,tracing]
+---
+
+Istio provides advanced capability to configure tracing options as of Istio 1.6.
+Previously, the only customization for tracing was the amount of randomly sampled requests.
+All of these are considered experimental.
+
+## Create a MeshConfig with trace settings
+
+All tracing options are configured by using MeshConfig during Istio installation.
+To simplify configuration, create a single YAML file to pass to `istioctl`.
+
+{{< text yaml >}}
+cat <<'EOF' > tracing.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    defaultConfig:
+      tracing:
+EOF
+{{< /text >}}
+
+Then, you can append any configuration options to the `tracing.yaml` file.
+
+## Trace sampling
+
+Istio captures a trace for all requests by default when installing with the demo profile.
+For example, when using the Bookinfo sample application, every time you access
+`/productpage` you see a corresponding trace in the
+dashboard. This sampling rate is suitable for a test or low traffic
+mesh. For a high traffic mesh you can lower the trace sampling
+percentage in one of two ways:
+
+{{ warning }}
+Previously, the recommended method was to change the `values.pilot.traceSampling` setting during the mesh setup
+ or to change the `PILOT_TRACE_SAMPLE` environment variable in the pilot or istiod deployment.
+While this method to alter sampling continues to work, the following method
+is strongly recommended instead.
+
+In the event that both are specified, the value specified in the `MeshConfig` will override any other setting.
+{{ /warning }}
+
+To modify the default random sampling, which is defaulted to a value of 1, add the following to your
+`tracing.yaml` file.
+
+{{< text yaml >}}
+        sampling: <VALUE>
+{{< /text >}}
+
+Where the `<VALUE>` should be in the range of 0.0 to 100.0 with a precision of 0.01.
+
+## Customizing tracing tags
+
+{{ warning }}
+The ability to add tracing tags to spans works for sidecar proxies and gateways.
+If you are concerned about rogue external client requests we do not recommend you
```

Comment:
Should we add the reasoning @nrjpoddar added here https://github.com/istio/istio/pull/23098#discussion_r413183241? I'm concerned that it may be too technical, but the implications are quite severe for end users. This is considered experimental, but I do not want users to have a bad experience.

Reply:
Let's remove the warning, it applies broadly for tracing in gateways.